agent/taskresource/firelens/firelensconfig_unix.go

```diff
@@ -56,6 +55,9 @@ const (
 
 	// socketPath is the path for socket file.
 	socketPath = "/var/run/fluent.sock"
+
+	// fluentTagOutputFormat is the format for the log tag, which is container name with "-firelens" appended
+	fluentTagOutputFormat = "%s-firelens*"
```

Comment:
why did we make this less specific

Reply:
See the description of the PR- it makes it so that later on customers can send logs to these task def configured outputs using the fluent logger SDKs. If the output matches the task ID, then their code would have to know the task ID (which is possible but annoying).